sklearn/gp_search.py

```diff
@@ -0,0 +1,421 @@
+"""
```

Comment:
Shouldnt this exist under the `grid_search` module, as for RandomizedSearchCV which is there with GridSearchCV? After all, this can be viewed as a smart exploration of a (possibly infinite) grid.